test/fgpu/test_engine.py

```diff
@@ -844,6 +844,12 @@ async def test_missing_heaps(
             if p and i + middle < len(dst_present):
                 x = out_data[:, i * spectra_per_heap : (i + 1) * spectra_per_heap]
                 y = out_data[:, (i + middle) * spectra_per_heap : (i + middle + 1) * spectra_per_heap]
+                # For narrowband they're not guaranteed to be equal, as the mixer
+                # phase varies over time. For this test the time difference is a
+                # multiple of the mixer wavelength, but there can still be
+                # numerical rounding differences. For the fixed seed chosen
+                # above, we get exact equality, but this test may need to be
+                # related slightly in future.
```

Comment:
"related"? I suspect you might have meant something else.

Reply:
Oops, was meant to be "relaxed".